packages/vite/src/node/plugins/resolve.ts

```diff
@@ -838,12 +838,21 @@ function resolveDeepImport(
   }
 
   if (relativeId) {
-    const resolved = tryFsResolve(
+    let resolved
+    resolved = tryFsResolve(
```

Comment:
We don't want implicit inferred any types

```suggestion
    let resolved = tryFsResolve(
```

Reply:
updated